hack/make/.integration-test-helpers

```diff
@@ -80,15 +81,21 @@ run_test_integration_suites() {
 			# Finally, we use periods as separator (instead of slashes) to be more
 			# in line with Java package names (which is what junit.xml was designed for)
 			pkgname="$(go env GOARCH).${pkgname//\//.}"
-			echo "Running $PWD (${pkgname}) flags=${flags}"
+
+			pkgtestflags=$flags
+			if [ -n "${TESTCOVERAGE}" ]; then
+				pkgtestflags="$pkgtestflags -test.coverprofile=${ABS_DEST}/${pkgname//./-}-coverage.out"
```

Comment:
Do we need changes to `.gitignore` and `.dockerignore`? https://github.com/moby/moby/blob/067062129171dba592638e8e3a0efbfa8a71376b/.gitignore#L23

Reply:
Not related I think, I don't see any reference to a `profile.out` file in master so I think it's just an orphan entry. And we don't need to coverage is pushed to `buundles`